app/helpers/application_helper.rb

```diff
@@ -296,4 +296,22 @@ def admin_config_description(content)
   def role_display_name(role)
     role.name.titlecase
   end
+
+  def render_tag_link(tag, filled: false, monochrome: false, classes: "")
+    color = tag_colors(tag)[:background] || Settings::UserExperience.primary_brand_color_hex
```

Comment:
Ruby hashes have a method called `fetch` which allows for providing a default value. The reason we're choosing a block here is that this default value has a side-effect (a cache or DB read) so we only want to execute it when `background` returns `nil`:

```suggestion
    color = tag_colors(tag).fetch(:background) { Settings::UserExperience.primary_brand_color_hex }
```

Note: your current code works fine too, just using this as an opportunity to share some Ruby concepts.

Reply:
Oh thanks, I appreciate your explanation! #TIL!

Reply:
reverted as it broke the build